themes/default/content/blog/announcing-kubernetes-helm-release-resource-preview/index.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Announcing Preview for Helm Release Resource for Kubernetes
+h1: "Announcing Preview for Helm Release Resource for Kubernetes"
+date: "2021-09-07"
+meta_desc: "Today we are announcing preview of a new Helm Release resource for Kubernetes which adds support for Helm natively on Pulumi."
+meta_image: "helm-release.gif"
+authors: ["vivek-lakshmanan"]
+tags: ["Kubernetes", "helm"]
+---
+
+Kubernetes has been a significant focus of Pulumi since its very beginnings. Pulumi added support for installing [Helm charts](https://helm.sh/) way back in [2018](https://www.pulumi.com/blog/using-helm-and-pulumi-to-define-cloud-native-infrastructure-as-code/) and it has seen significant adoption by users since. However, Pulumi's current Chart integration lacks support for some increasingly common advanced features in Helm charts, e.g.:
```

Comment:
The blog quote and other links within pulumi.com should likely be relrefs - see other posts for the syntax.

Reply:
I have updated it. Could you take a look through to make sure I miss anything? Looks like since the pulumi-hugo/docs split, the relrefs across to`/docs/` etc are not retained locally (understandably). @cnunciato - will the previews reflect them correctly? (Will check once the PR preview is rendered).

Reply:
@viveklak Sorry, bit of a late response, but yes, `relref` should work both locally and in PR previews for links pointing to content that resides in this repo, so since the blog is in this repo, that'll work. (And your preview seems good.)

Reply:
> @viveklak Sorry, bit of a late response, but yes, `relref` should work both locally and in PR previews for links pointing to content that resides in this repo, so since the blog is in this repo, that'll work. (And your preview seems good.)

Chatted with @cnunciato offline and the relrefs to docs are expected to be broken in the preview in this repo. The merge preview on docs should render the right links.